### PR TITLE
Initial support for 1.16.210

### PR DESCRIPTION
**Pull request by Gabriel8579**
Needs further investigation to see if something has been affected

### CODE REVIEW

**Comment by RORIdev** [comment]
Heya, always remember to `git fetch` and `git pull --rebase` after doing any changes.

**Comment by PhanaticD** [comment]
tested this out, I just instantly disconnect

File: src/main/java/cn/nukkit/network/protocol/AvailableCommandsPacket.java [hunk 1]
```diff
@@ -25,23 +26,23 @@
...
-    public static final int ARG_TYPE_FILE_PATH = 14;
-
-    public static final int ARG_TYPE_STRING = 29;
-    public static final int ARG_TYPE_BLOCK_POSITION = 37;
-    public static final int ARG_TYPE_POSITION = 38;
-
-    public static final int ARG_TYPE_MESSAGE = 41;
-    public static final int ARG_TYPE_RAWTEXT = 43;
-    public static final int ARG_TYPE_JSON = 47;
-    public static final int ARG_TYPE_COMMAND = 54;
```
Comment (joserobjr) [inline src/main/java/cn/nukkit/network/protocol/AvailableCommandsPacket.java:44]: This breaking change will be solved at https://board.powernukkit.org/youtrack/issue/PN-1000

**Review by joserobjr — DISMISSED** [review]
The code looks good, thank you! :)

**Comment by joserobjr** [comment]
I got a testing feedback:

https://discord.com/channels/728280425255927879/728280425255927882/820910007981047849

> 🧙♂  Severus Snape 🧙♂
> Seems like all works fine.
> But you cant get items out of Itemframes.
> Only in Gamemode 1 it works
> And witherskeleton eggs are twice in creative.
> But both eggs spawns skeleton
> At next, dragoneggs can't break in Gm0 and Gm1

**Comment by joserobjr** [comment]
We also need to make a JUnit to test if the removal of the `deprecated` block property from the stripped logs could cause any problem to existing worlds (if somebody got a `minecraft:stripped_crimson_hyphae;deprecated=1;pillar_axis=y` for example, it should be converted to `minecraft:stripped_crimson_hyphae;pillar_axis=y` without rising console errors)

**Pull request merged**